Lib/fontParts/base/image.py

```diff
@@ -26,6 +27,24 @@ def _reprContents(self):
             contents += self.glyph._reprContents()
         return contents
 
+    # Add a nonzero/bool method for testing if there is image data
+    # otherwise True is always returned
+    PY3 = sys.version_info[0] == 3
+    PY2 = sys.version_info[0] == 2
```

Comment:
as fontTools is already required: use `from fontTools.misc.py23 import PY2, PY3`

I like to have cleaner code and some additional import that fixes compatibility